src/App.tsx

```diff
@@ -1,61 +1,88 @@
 import './App.scss';
+import React, { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { UserInfo } from './components/UserInfo';
+import { Todo } from './types/types';
+import { TodoList } from './components/TodoList';
+import { TodoInfo } from './components/TodoInfo';
 
 export const App = () => {
+  const [todos, setTodos] = useState(todosFromServer);
+  const [title, setTitle] = useState<string>('');
+  const [errorTitle, setErrorTitle] = useState(false);
+  const [user, setUser] = useState(0);
+  const [errorUser, setErrorUser] = useState(false);
+
+  const resetInputs = () => {
+    setTitle('');
+    setUser(0);
+
+    setErrorTitle(false);
+    setErrorUser(false);
```

Comment:
consider to reset error in onChange event for proper input

Reply:
https://github.com/mate-academy/react_add-todo-form/pull/1902/files#r1275938825
this comment not fixed.
please pay attention on all comments in next tasks)